corehq/apps/data_dictionary/management/commands/populate_case_prop_groups.py

```diff
@@ -31,3 +33,17 @@ def populate_case_prop_groups(domain):
         )
         case_prop.group_obj = group
         case_prop.save()
+
+
+def remove_out_of_sync_prop_and_groups():
+    # Remove properties have different group and group_obj fields
+    properties_out_of_sync = (CaseProperty.objects
+                            .filter(group_obj__isnull=False)
+                            .filter(~Q(group_obj__name=F('group'))))
+    print("Remove out of sync groups for {} properties".format(len(properties_out_of_sync)))
```

Comment:
Good to also show the name of these case properties or some sort of representation so that user can actually check if things are okay. 
Same for the logging later as well

Reply:
Fixed. [c41883e](https://github.com/dimagi/commcare-hq/pull/32799/commits/c41883e62716997754992920b8be35c42ce7b62d)